examples/xgboost-comprehensive/xgboost_comprehensive/server_app.py

```diff
@@ -59,45 +76,88 @@ def evaluate_fn(
                 iteration=bst.num_boosted_rounds() - 1,
             )
             auc = round(float(eval_results.split("\t")[1].split(":")[1]), 4)
-            log(INFO, f"AUC = {auc} at round {server_round}")
 
             return 0, {"AUC": auc}
 
     return evaluate_fn
 
 
-class CyclicClientManager(SimpleClientManager):
-    """Provides a cyclic client selection rule."""
+def evaluate_metrics_aggregation(eval_metrics):
+    """Return an aggregated metric (AUC) for evaluation."""
+    total_num = sum([num for num, _ in eval_metrics])
+    auc_aggregated = (
+        sum([metrics["AUC"] * num for num, metrics in eval_metrics]) / total_num
+    )
+    metrics_aggregated = {"AUC": auc_aggregated}
```

Comment:
potential place to save to disk?

Reply:
In this function, we can't get the current round number to be saved. To do that, we have to override the `aggregate_evaluate` function of the strategy. It may be a bit too redundant for this example imo. I'd not have save_to_disk for this function. @jafermarq Wdyt?

Reply:
ok. makes sense. Maybe add a brief comment in the README.md that points the reader to the advanced-pytorch example showing how strategies could be extended?